h2o-docs/src/product/automl.rst

```diff
@@ -10,27 +10,188 @@ H2O's AutoML can be used for automating the machine learning workflow, which inc
 AutoML Interface
 ----------------
 
-The AutoML interface is designed to have as few parameters as possible so that all the user needs to do is point to their dataset, identify the response column and optionally specify a time-constraint.  
+The AutoML interface is designed to have as few parameters as possible so that all the user needs to do is point to their dataset, identify the response column and optionally specify a time-constraint. 
 
-Example in R
-~~~~~~~~~~~~
+In both the R and Python API, AutoML uses the same data-related arguments, `x, y, training_frame, validation_frame`, as the other H2O algorithms.  
 
-Here’s an R code example showing basic usage of the ``h2o.automl()`` function:
+- The `x` argument only needs to be specified if the user wants to exclude predictor columns from their data frame.  If all columns (other than the response) should be used in prediction, this can be left blank/unspecified.
+- The `validation_frame` argument is optional and will be used for early stopping within the training process of the individual models in the AutoML run.  
+- The `test_frame` argument allows the user to specify a particular data frame to rank the models on the leaderboard.  This frame will not be used for anything besides creating the leaderboard.
+- To control how long the AutoML run will execute, the user can specify `max_runtime_secs`, which defaults to 600 seconds (10 minutes).
 
-::
+
+Code Examples
+~~~~~~~~~~~~~
+
+Here’s an example showing basic usage of the ``h2o.automl()`` function in *R* and the ``H2OAutoML`` class in *Python*:
+
+.. example-code::
+   .. code-block:: r
 
     library(h2o)
+
     h2o.init()
 
-    df <- h2o.importFile("http://s3.amazonaws.com/h2o-public-test-data/smalldata/prostate/prostate.csv.zip")
-    df$CAPSULE <- as.factor(df$CAPSULE)
-    aml <- h2o.automl(x = 3:8, y = 2, training_frame = df)
+    # Import a sample binary outcome train/test set into H2O
+    train <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_train_10k.csv")
+    test <- h2o.importFile("https://s3.amazonaws.com/erin-data/higgs/higgs_test_5k.csv")
+
+    # Identify predictors and response
+    y <- "response"
+    x <- setdiff(names(train), y)
+
+    # For binary classification, response should be a factor
+    train[,y] <- as.factor(train[,y])
+    test[,y] <- as.factor(test[,y])
+
+    aml <- h2o.automl(x = x, y = y, 
+                      training_frame = train,
+                      test_frame = test,
+                      max_runtime_secs = 30)
 
     # View the AutoML Leaderboard
-    print(aml@leaderboard)
+    aml@leaderboard
+
+    #                                              model_id      auc
```

Comment:
Might need to update the output since I added logloss for binomial and rmse,mae, and rmsle for regression